cli/keys.go

```diff
@@ -164,3 +162,22 @@ func KeysCmd(prompt prompts.Prompt) *cobra.Command {
 	cmd.Flags().BoolVarP(&flags.install, "install", "i", false, "Install dependencies if not installed without asking")
 	return cmd
 }
+
+func saveMnemonic(mnemonic string) error {
+	file, err := ioutil.TempFile(os.TempDir(), "sedge_mnemonic")
+	if err != nil {
+		return err
+	}
+	defer os.Remove(file.Name())
+	if _, err := file.WriteString(fmt.Sprintf("Mnemonic:\n\n%s\n\n", mnemonic)); err != nil {
```

Comment:
It would be great to show here a constant message advising the user to write down the mnemonic and instructions on how to exit the text editor.  A good reference for a message here is what the [staking-deposit-cli](https://github.com/ethereum/staking-deposit-cli/blob/76ed78224fdfe3daca788d12442b3d1a37978296/staking_deposit/intl/en/cli/new_mnemonic.json) tool shows.

Reply:
Done -> https://github.com/NethermindEth/sedge/pull/169/commits/8d61d61116c955b69b2449c9a0555162543920da